src/test/java/reactor/netty/http/client/HttpClientProxyTest.java

```diff
@@ -165,13 +170,52 @@ public void testIssue804() {
 				    .verify(Duration.ofSeconds(30));
 	}
 
-	private Mono<Tuple2<String, HttpHeaders>> sendRequest(
+    @Test
+    public void shouldNotResolveTargetHostnameWhenMetricsEnabled() {
```

Comment:
nit: use tabs not spaces

Reply:
@violetagg I was trying to be careful with the tabs overall, but apparently I missed it in a couple of places.  All should have been addressed with the recent commits.